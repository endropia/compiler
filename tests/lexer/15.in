$7FFFFFFF
$80000000