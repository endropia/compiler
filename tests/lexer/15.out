1	1	Integer	2147483647	$7FFFFFFF
Overflow integer